src/hotspot/cpu/riscv/templateTable_riscv_64.cpp

```diff
@@ -1494,82 +1503,80 @@ void TemplateTable::convert() {
   Label done;
   switch (bytecode()) {
```

Comment:
Hard code. @vasyoid @DmKazakov please check it, you are more familiar with such PPC instructions